showcase/styles.css

```
@@ -0,0 +1,203 @@
+  --surface: #141416;
+  --surface-raised: #1a1a1d;
+  --border: #252528;
+  --border-muted: #1c1c1f;
+  --text-primary: #e4e4e7;
+  --text-secondary: #a1a1aa;
+  --text-muted: #71717a;
+  --accent: #f59e0b;
+  --accent-muted: #92400e;
+  --teal: #0d9488;
+  --teal-muted: #115e59;
+  --font-mono: ui-monospace, 'SF Mono', 'Cascadia Code', 'JetBrains Mono', Menlo, monospace;
```

Stylelint (17.14.0)

[error] Expected "Menlo" to be "menlo" (value-keyword-case)

(value-keyword-case)